run-cis-benchmark.sh

```diff
@@ -90,235 +123,680 @@ if [[ $(whoami) != "root" ]]; then
         echo "You must run this script as root"
         exit 1
 fi
-
 echo "CIS CentOS Linux 7 Benchmark (v2.2.0 - 12-27-2017)"
 echo
 echo hostname : `hostname`
 echo time     : `date`
 echo =================================================================================
+if [[ $(test_filter $1 1.1.1.1) == 0 ]]; then
 test_wrapper 1.1.1.1 "Ensure mounting of cramfs filesystems is disabled (Scored)" Yes Server1 Workstation1
+fi
+if [[ $(test_filter $1 1.1.1.2) == 0 ]]; then
 test_wrapper 1.1.1.2 "Ensure mounting of freevxfs filesystems is disabled (Scored)" Yes Server1 Workstation1
```

Comment:
Make sure to 2-space indent anything within a block (if, while, for, functions, etc.)